### PR TITLE
List specific areas for improvement as identified by Bazel Vision doc…

### PR DESCRIPTION
**Pull request by dslomov**
…ument.

Remove outdated 1.0 roadmap. We should update this page once we finalize the 1.0 plan.

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -3,88 +3,31 @@ layout: contribute
...
+    _gradual adoption_ of Bazel.
+*   [**Configurability**](roadmaps/configuration.html). Outside of a few
+    hardcoded cases (fat apk for Android,host -> target transitions), the promise
+    of single-command multi-platform builds is unfulfilled. 
+*   **Skylark API and tooling**. We need to support the rule authors with 
+    **clear, documented, stable API**, and **strong language and tooling**:
+    documentation generators, refactoring tools, testing frameworks and so on.
+
+## Roadmaps for specific Bazel areas
+*  [Configuarbility](roadmaps/configuration.html)
```
Comment (jin) [inline roadmap.md]: typo in Configurability

Comment (dslomov) [reply roadmap.md]: Fixed

File: roadmap.md [hunk 2]
```diff
@@ -3,88 +3,31 @@ layout: contribute
...
-* P1. Support policy is defined regarding LTS release
-* P1. Bazel respects the standard for Debian packaging
-* P2. Bazel is in the list of Debian package for the next stable
-
+# Bazel Roadmap
+
+## Bazel Core Improvement Areas 
+
+As of July 2018, critical areas that Bazel team addresses within Bazel core to
+fulfill the [Bazel Vision](https://docs.bazel.build/versions/master/bazel-vision.html)
```
Comment (laurentlb) [inline roadmap.md]: link doesn't exist?

File: roadmap.md [hunk 3]
```diff
@@ -3,88 +3,31 @@ layout: contribute
...
+*   [**Configurability**](roadmaps/configuration.html). Outside of a few
+    hardcoded cases (fat apk for Android,host -> target transitions), the promise
+    of single-command multi-platform builds is unfulfilled. 
+*   **Skylark API and tooling**. We need to support the rule authors with 
+    **clear, documented, stable API**, and **strong language and tooling**:
+    documentation generators, refactoring tools, testing frameworks and so on.
+
+## Roadmaps for specific Bazel areas
+*  [Configurability](roadmaps/configuration.html)
+*  [Windows](roadmaps/windows.html)
```
Comment (laurentlb) [inline roadmap.md]: With this change, I suggest we get rid of the menu on the left (keep only the link to the general roadmap, not the subsections).

File: roadmap.md (hunk 2 above)
Comment (dslomov) [reply roadmap.md]: https://github.com/bazelbuild/bazel/pull/5612. I'll only merge once that is merged.

File: roadmap.md (hunk 3 above)
Comment (laurentlb) [reply roadmap.md]: Add links to: skylark, external-deps, platforms

**Pull request merged**